apps/discovery_api/lib/discovery_api_web/router.ex

```diff
@@ -30,25 +30,16 @@ defmodule DiscoveryApiWeb.Router do
 
   pipeline :reject_cookies_from_ajax do
     plug(DiscoveryApiWeb.Plugs.SetAllowedOrigin)
-    plug(DiscoveryApiWeb.Plugs.CookieMonster)
   end
 
   pipeline :global_headers do
     plug(DiscoveryApiWeb.Plugs.NoStore)
   end
 
-  scope "/api/v1", DiscoveryApiWeb do
-    pipe_through([:reject_cookies_from_ajax, :global_headers])
-
-    get("/login", LoginController, :login)
-  end
-
   scope "/api/v1", DiscoveryApiWeb do
     pipe_through([:reject_cookies_from_ajax, :verify_token, :ensure_authenticated, :global_headers])
 
     post("/logged-in", UserController, :logged_in)
-
-    get("/logout", LoginController, :logout)
```

Comment:
Doesn't this logout endpoint work for auth0 as well? And if it does, don't we need this or is there another way to logout aside from clicking the login zone link?

Reply:
sweet